Makefile

```diff
@@ -17,7 +17,7 @@ POLYFILL_SRC = \
   src/runtime/polyfills/WeakSet.js \
   src/runtime/polyfills/polyfills.js
 
-RUNTIME_MODULES = src/runtime/runtime-modules.js
+RUNTIME_MODULES = src/runtime/runtimeModules.js
```

Comment:
Based on your recent commits I thought you wanted to rename in this direction. Does not matter to me, but I'd prefer one or the other.